configure

```diff
@@ -954,18 +960,73 @@ sub targets_win64 {
   return $arch eq 'x64';
 }
 
+sub system_default_install_dir {
+  if (is_windows()) {
+    return targets_win64() ? $ENV{'ProgramFiles'} : $ENV{'ProgramFiles(x86)'};
```

Comment:
Isn't there usually a space between "ProgramFiles" and "(x86)"? Although maybe that's just the environment variable and not the path. More importantly, we need to make sure we're supporting win32 arch, which is still windows, but is not 64 bit, and won't have the (x86) stuff in it.